Demo for image appending

A-Cain opened the pull request:
Two new classes added.
AddImage and TestAddImage
The former will take a pdf that I placed in the appended resoruces folder and spit out the same pdf with an image appended

A-Cain committed:
AddImage.java's main function will create a version of examplePDFWithText.pdf with pamcamke.jpg appended when it is run. I do not know why the image is upside down.

A-Cain committed:
AddImage.java's main function will create a version of examplePDFWithText.pdf with pamcamke.jpg appended when it is run.
All needed files and output are stored in AddImageResources directory in tools folder

Nerd1000percent commented on tools/src/test/java/org/apache/pdfbox/tools/imageio/TestAddImage.java:
```diff
@@ -0,0 +1,45 @@
+public void testLoadFileAndInitializeStream() throws IOException {
+
+    AddImage add = new AddImage();
+    String expected = "C:/Users/Asher/Downloads/itext7start/itext7start/mavenproject2/results/chapter01/hello.pdf";
```
Can that be changed based on the user? Im assuming that is the location of PDF file created.

Nerd1000percent replied:
Will that fail if ran on another PC?

A-Cain replied:
That should be an outdated version, the latest version has a path to somewhere within the project directory

A-Cain replied:
If you can find the commit I made dated today (since I made it after midnight), that should have the up to date code

A-Cain committed:
AddImage.java's main function will create a version of examplePDFWithText.pdf with pamcamke.jpg appended when it is run.
All needed files and output are stored in AddImageResources directory in tools folder
